zulipterminal/model.py

```diff
@@ -183,8 +182,8 @@ def __init__(self, controller: Any) -> None:
 
         # NOTE: The date_created field of stream has been added in feature
         # level 30, server version 4. For consistency we add this field
-        # on server iterations even before this with value of None.
-        if self.server_feature_level is None or self.server_feature_level < 30:
+        # on server iterations even before this with value of 0.
+        if self.server_feature_level == 0 or self.server_feature_level < 30:
```

Comment:
This original comment is not great, but this is a good time to adjust to say for earlier versions (in general, not just None or zero).